doc/source/whatsnew/v0.19.0.txt

```diff
@@ -437,8 +437,7 @@ API changes
 - ``pd.Timedelta(None)`` is now accepted and will return ``NaT``, mirroring ``pd.Timestamp`` (:issue:`13687`)
 - ``Timestamp``, ``Period``, ``DatetimeIndex``, ``PeriodIndex`` and ``.dt`` accessor have gained a ``.is_leap_year`` property to check whether the date belongs to a leap year. (:issue:`13727`)
 - ``pd.read_hdf`` will now raise a ``ValueError`` instead of ``KeyError``, if a mode other than ``r``, ``r+`` and ``a`` is supplied. (:issue:`13623`)
-
-
+- ``groupby()`` now accepts ``level=[0]`` (in addition to ``level=0``), ``level=-1`` and ``level=[-1]`` for non-``MultiIndex``ed objects. (:issue:`13907`)
```

Comment:
too complicated. a user is reading this. just say ``.groupby()`` will accept a scalar and a single element list for specifyig level on an Index grouper.